core/src/main/java/com/homeaway/streamplatform/streamregistry/StreamRegistryApplication.java

```diff
@@ -261,8 +261,13 @@ private ManagedKStreams createManagedKStreams(final StreamRegistryConfiguration
         configuration.getKafkaStreamsConfig().getKstreamsProperties().forEach(kstreamsProperties::put);
         kstreamsProperties.put(ROCKSDB_CONFIG_SETTER_CLASS_CONFIG, CustomRocksDBConfig.class);
         TopicsConfig topicsConfig = configuration.getTopicsConfig();
+        SchemaManagerConfig schemaManagerConfig = configuration.getSchemaManagerConfig();
+
+        Preconditions.checkState(schemaManagerConfig.getProperties() != null
+                && schemaManagerConfig.getProperties().containsKey(SCHEMA_REGISTRY_URL_CONFIG),
+            "schemaManagerConfig properties must define schema.registry.url");
```

Comment:
Reuse the constant: `must define " + SCHEMA_REGISTRY_URL_CONFIG`

Reply:
fair.